docs/fundamentals/jcloud/advanced.md

```diff
@@ -73,6 +73,47 @@ executors:
           type: efs
 ```
 
+### Gateway
+
+For the internet exposure, JCloud now provides support for [Kong gateway](https://konghq.com/products/api-gateway-platform). 
+
+We currently support 2 types of gateway.
+
+#### `alb`
+
+At current stage, `alb` is default gateway type due to backward compatibility. `alb` has below features, [docs] (https://aws.amazon.com/elasticloadbalancing/application-load-balancer/)
+
+- Use AWS signed public certs
+
+####  `Kong`
+
+`Kong` gateway is current working as ingress controller. We haven't enable other features yet. [docs] (https://docs.konghq.com/kubernetes-ingress-controller/latest/)
+
+- Use let's encrypt signed widecard certs
+- Use nginx to distribute traffic as ingress controller
+
+```{tip}
+Normally, user can retrieve the CA from client end. The difference of cert CA usually will not affect the jina usage unless the CA is not in the trust list.
```

Comment:
What is `CA`? Usually we try to spell out acronyms at least once before we use them, so inexperienced users (like myself) know what's going on